pkgs/development/python-modules/pyicumessageformat/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pyicumessageformat";
+  version = "1.0.0";
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  src = fetchPypi {
+    pname = "pyicumessageformat";
+    inherit version;
```

Comment:
```suggestion
    inherit pname version;
```

Reply:
The motivation for this was https://github.com/nix-community/nixpkgs-lint/issues/21, it got addressed already in an earlier review.

Reply:
It's different for `fetchPypi`.

Reply:
How is it different?

Reply:
`repo = pname` is accidental but the `pname` of a Python package is supposed to be exactly what it's called on PyPI.

Reply:
https://github.com/NixOS/nixpkgs/issues/277994#issuecomment-2318421051

Reply:
Makes sense to me, I'll change that.